hr_holidays_leave_overlap/models/hr_holidays.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2018 Onestein (<http://www.onestein.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from datetime import timedelta
+
+
+class HrHolidays(models.Model):
+    _inherit = 'hr.holidays'
+
+    @api.model
+    def create(self, values):
+        if values['type'] == 'remove':
+
+            including = self.search([
+                ('date_from', '<', values['date_from']),
+                ('date_to', '>', values['date_to']),
+                ('employee_id', '=', values['employee_id']),
```

Comment:
Please fix this! In case holiday_type='category',  the employee_id could be not present in the values.
Check also the rest of the code.

Reply:
Still this point is left